measurements/config.py

```diff
@@ -29,3 +32,10 @@
 
 # As of 2017-07-18 635830 is the latest index in the database
 REPORT_INDEX_OFFSET = int(os.environ.get("REPORT_INDEX_OFFSET", "635830"))
+
+REQID_HDR = "Request-Id"
```

Comment:
You should probably use some more generic header to avoid possible clashes. For example `X-OONI-Request-Id`

Reply:
[Generic](https://devcenter.heroku.com/articles/http-request-id) one is `X-Request-Id`, but it's not generic enough as noone had bothered to [register it with IANA](http://www.iana.org/assignments/message-headers/message-headers.xhtml). :-) Also, [`X-` prefix is deprecated by RFC6648](https://tools.ietf.org/html/rfc6648).

Anyway, I intentionally use generic one and [filter it on ingress](https://github.com/TheTorProject/ooni-sysadmin/blob/6e65c46af2ec3f742e87b167566ee6fc5381d723/ansible/roles/nginx-stable/templates/nginx.conf#L17-L33) as it has to be filtered anyway.

Reply:
But that's good catch, I'd rather use `X-Request-ID` that is even more generic.